php/laravel-s-laravel/composer.json

```
@@ -0,0 +1,11 @@
+{
+  "require": {
+    "hhxsv5/laravel-s": "~3.7",
+    "laravel/framework": "~8.0"
```

sorry I had probably misled you. the idea is here to promote compare `laravel-s`. the the important version is `3.7`, no matter in which version `laravel` is
```suggestion
  "require": {
    "hhxsv5/laravel-s": "~3.7.0",
    "laravel/framework": "*"
```

---

It's okay, I get u.